Makefile

```diff
@@ -85,14 +85,17 @@ binary: build
 
 build: bundles
 ifeq ($(DOCKER_OSARCH), linux/arm)
-	# A few libnetwork integration tests require that the kernel be
-	# configured with "dummy" network interface and has the module
-	# loaded. However, the dummy module is not available by default
-	# on arm images. This ensures that it's built and loaded.
-	echo "Syncing kernel modules"
-	oc-sync-kernel-modules
-	depmod
-	modprobe dummy
+	@bash -c 'if [ "`lsmod | grep dummy`" = "" ]; then \
+	  echo "A few libnetwork integration tests require that the kernel be"; \
+	  echo "configured with "dummy" network interface and has the module"; \
+	  echo "loaded. However, the dummy module is not loaded."; \
+	  echo "Please load the kernel module permanently."; \
+	  echo "Syncing kernel modules"; \
+	  oc-sync-kernel-modules; \
```

Comment:
I think this is a Scaleway only tool. I left that inside as it seems that the ARM builder needed that. But this can also be done while provisioning new machines in Scaleway.